builtin/aws/lambda/mapper.go

```diff
@@ -0,0 +1,19 @@
+package lambda
```

Comment:
Could you move this to `utils.go` instead? Mappers are a specific argmapper concept that are automatically called and have to be registered. This appears to just be a helper function to convert one string to another.

Reply:
TIL! Ok, renamed this `mapper.go` file `utils.go`